### DISCUSSION TITLE
how to filter outgoing email with subject wise

### DISCUSSION THREAD

**Discussion by rccess** [postalserver/postal#1779] [Help with using Postal]
Hello team,

how to filter outgoing email with **subject** wise

![image](https://user-images.githubusercontent.com/32220165/147078967-e82a40f7-9796-4d77-b4cd-a7460680268b.png)

Please do advise

**Comment by brywhi** [comment]
I don't think this is possible currently- but it would be an awesome feature. However, I think you should be able to find it using SQL:
1. Connect to your SQL db server. If using Docker as instructed in the install instructions, do `docker exec -it postal-mariadb bash`
2. Launch mysql `mysql -u root -p`
3. Find the id of the server you want to search using SQL: `use postal; select id from servers where name = 'My Awesome Mail Server';`
4. Access that server's database and query its message table. Ex, for server 1: `use postal-server-1; select * from messages where subject like 'Success- Internal%' limit 10;`
